internal/formatter/builders/css/core/CSSPercentage.ts

```diff
@@ -1,9 +1,9 @@
 import {CSSPercentage} from "@internal/ast";
-import {Builder, Token} from "@internal/formatter";
+import {Builder, Token, concat} from "@internal/formatter";
 
 export default function CSSPercentage(
 	builder: Builder,
 	node: CSSPercentage,
 ): Token {
-	throw new Error("unimplemented");
+	return concat([String(node.value), "px"]);
```

Comment:
I think the correct implementation should be `%`
```suggestion
	return concat([String(node.value), "%"]);
```

Also, we should implement a test, so we should add something like:
```css
width: 80%;
```

Reply:
@ematipico Yes. right I found that while double check. Thanks 👍